From: tcamuso@redhat.com <tcamuso@redhat.com>
Date: Mon, 7 Jan 2008 22:15:21 -0500
Subject: [x86] mmconfig: remove platforms from the blacklist
Message-id: 20080108031521.15008.95362.sendpatchset@dhcp83-188.boston.redhat.com
O-Subject: [PATCH 1/6][Version-2]PCI: x86 MMCONFIG: Remove all but one platform from the blacklist
Bugzilla: 239673 253288 408551

diff --git a/arch/i386/pci/mmconfig.c b/arch/i386/pci/mmconfig.c
index 8fd4c92..37bd283 100644
--- a/arch/i386/pci/mmconfig.c
+++ b/arch/i386/pci/mmconfig.c
@@ -202,27 +202,6 @@ static int __devinit disable_mmconf(struct dmi_system_id *d)
 static struct dmi_system_id __devinitdata nommconf_dmi_table[] = {
 	{
 		.callback = disable_mmconf,
-		.ident = "HP xw9300 Workstation",
-		.matches = {
-			DMI_MATCH(DMI_PRODUCT_NAME, "HP xw9300 Workstation"),
-		},
-	},
-	{
-		.callback = disable_mmconf,
-		.ident = "HP xw9400 Workstation",
-		.matches = {
-			DMI_MATCH(DMI_PRODUCT_NAME, "HP xw9400 Workstation"),
-		},
-	},
-	{
-		.callback = disable_mmconf,
-		.ident = "ProLiant DL585 G2",
-		.matches = {
-			DMI_MATCH(DMI_PRODUCT_NAME, "ProLiant DL585 G2"),
-		},
-	},
-	{
-		.callback = disable_mmconf,
 		.ident = "HP Compaq dc5700 Microtower",
 		.matches = {
 			DMI_MATCH(DMI_PRODUCT_NAME, 
diff --git a/arch/x86_64/pci/mmconfig.c b/arch/x86_64/pci/mmconfig.c
index 5c9ebb4..375f565 100644
--- a/arch/x86_64/pci/mmconfig.c
+++ b/arch/x86_64/pci/mmconfig.c
@@ -177,27 +177,6 @@ static int __devinit disable_mmconf(struct dmi_system_id *d)
  * Systems which cannot use PCI MMCONFIG at this time...
  */
 static struct dmi_system_id __devinitdata nommconf_dmi_table[] = {
-	{
-		.callback = disable_mmconf,
-		.ident = "HP xw9300 Workstation",
-		.matches = {
-			DMI_MATCH(DMI_PRODUCT_NAME, "HP xw9300 Workstation"),
-		},
-	},
-	{
-		.callback = disable_mmconf,
-		.ident = "HP xw9400 Workstation",
-		.matches = {
-			DMI_MATCH(DMI_PRODUCT_NAME, "HP xw9400 Workstation"),
-		},
-	},
-        {
-                .callback = disable_mmconf,
-                .ident = "ProLiant DL585 G2",
-                .matches = {
-                        DMI_MATCH(DMI_PRODUCT_NAME, "ProLiant DL585 G2"),
-                },
-        },
         {
                 .callback = disable_mmconf,
                 .ident = "HP Compaq dc5700 Microtower",
